pkg/providers/nutanix/client.go

```diff
@@ -0,0 +1,18 @@
+package nutanix
+
+import (
+	"context"
+
+	v3 "github.com/nutanix-cloud-native/prism-go-client/v3"
+)
+
+type client interface {
```

Comment:
This client is used in various places and nearly always referenced with a `v3Client` variable. Can we invert that and call the interface `v3Client` and the variables just `client`. This way, the emphasis is on the expected type, not the variable name.